docs_user/modules/proc_migrating-mon-from-controller-nodes.adoc

```diff
@@ -0,0 +1,411 @@
+`metalsmith.yaml` for the CephStorageNodes:
++
+[source,yaml]
+----
+  - name: CephStorage
+    count: 2
+    instances:
+      - hostname: oc0-ceph-0
+        name: oc0-ceph-0
+      - hostname: oc0-ceph-1
+        name: oc0-ceph-1
+    defaults:
+      networks:
+        - network: ctlplane
+          vif: true
+        - network: storage_cloud_0
+            subnet: storage_cloud_0_subnet
+        - network: storage_mgmt_cloud_0
+            subnet: storage_mgmt_cloud_0_subnet
+      network_config:
+        template: templates/single_nic_vlans/single_nic_vlans_storage.j2
+----
+
+. Run the provisioning command:
++
+----
+openstack overcloud node provision \
+  -o overcloud-baremetal-deployed-0.yaml --stack overcloud-0 \
+  --network-config -y --concurrency 2 /home/stack/metalsmith-0.yam
+----
+
+. Verify that the storage network is configured on the target nodes:
++
+----
+(undercloud) [stack@undercloud ~]$ ssh heat-admin@192.168.24.14 ip -o -4 a
+1: lo    inet 127.0.0.1/8 scope host lo\       valid_lft forever preferred_lft forever
+5: br-storage    inet 192.168.24.14/24 brd 192.168.24.255 scope global br-storage\       valid_lft forever preferred_lft forever
+6: vlan1    inet 192.168.24.14/24 brd 192.168.24.255 scope global vlan1\       valid_lft forever preferred_lft forever
+7: vlan11    inet 172.16.11.172/24 brd 172.16.11.255 scope global vlan11\       valid_lft forever preferred_lft forever
+8: vlan12    inet 172.16.12.46/24 brd 172.16.12.255 scope global vlan12\       valid_lft forever preferred_lft forever
+----
+
+.Procedure
+
+. Before you start the migration, ssh into the target node and enable the
+  firewall rules that are required to reach a Mon service:
++
+----
+for port in 3300 6789; {
+    ssh heat-admin@<target_node> sudo iptables -I INPUT \
+    -p tcp -m tcp --dport $port -m conntrack --ctstate NEW \
+    -j ACCEPT;
+}
+----
+
+* Replace `<target_node>` with the hostname of the node that is supposed to
+  host the new mon.
+
+. Check that the rules are properly applied and persist them:
++
+----
+sudo iptables-save
+sudo systemctl restart iptables
+----
+
+. To migrate the existing Mons to the target {Ceph} nodes, create the following
+  {Ceph} spec from the first mon (or the first controller) and modify the
+  placement based on the appropriate label.
++
+[source,yaml]
+----
+service_type: mon
+service_id: mon
+placement:
+  label: mon
+----
+
+. Save the spec in `/tmp/mon.yaml`
+. Apply the spec with cephadm by using the orchestrator:
++
+----
+sudo cephadm shell -m /tmp/mon.yaml
+ceph orch apply -i /mnt/mon.yaml
+----
++
+[NOTE]
+The effect of applying the `mon.yaml` spec is to normalize the existng placement
+strategy to use `labels` instead of `hosts`. By doing this any node with the `mon`
+label is able to host a Ceph mon daemon.
+You can execute the step above once to avoid multiple iterations over this
+procedure when multiple Ceph Mons are migrated.
```

Comment:
```suggestion
Applying the `mon.yaml` spec allows the existing placement
strategy to use `labels` instead of `hosts`. As a result, any node with the `mon`
label can host a Ceph mon daemon.
Execute this step once to avoid multiple iterations when multiple Ceph Mons are migrated.
```
@fmount Let me know if my edits changed the intended meaning of the note.

Reply:
it's good, thank you for the suggestion!